PyRDF/backend/Dist.py

```diff
@@ -6,41 +6,21 @@
 import warnings
 import ROOT
 import numpy
+import logging
+from collections import namedtuple
 
+logger = logging.getLogger(__name__)
 
-class Range(object):
-    """
-    Base class to represent ranges.
+Range = namedtuple("Range", ["start", "end", "filelist", "friend_info"])
```

Comment:
Why this has to be a named tuple? Later, you use it as a conventional tuple.

Reply:
Mmm I'm not sure I get what you mean. In this PR `Range`s are only created in `get_clustered_ranges`. Later in the file (lines 474-475) the `start` and `end` attributes of the named tuple are retrieved by the name, hence the need for a namedtuple. Previously `Range` was a full-fledged python class and the namedtuple provides the same functionality with a more lightweight container. 

And naturally it's also for aesthetic reasons :smile:

Reply:
Alright, I missed that you access the elements by name (was looking for it, but saw mainly indexing). I agree, it results in better readable code!